lxd/daemon.go

```diff
@@ -318,13 +318,13 @@ func StartDaemon(listenAddr string) (*Daemon, error) {
 
 	/* Create required paths */
 	d.lxcpath = shared.VarPath("lxc")
-	err = os.MkdirAll(shared.VarPath("/"), 0755)
+	err = os.MkdirAll(shared.VarPath("lxc"), 0755)
 	if err != nil {
 		return nil, err
 	}
 
 	// Create default directories
-	dirs := []string{"images", "lxc", "devlxd"}
+	dirs := []string{"images", "devlxd"}
```

Comment:
So this is wrong actually, devlxd is bind-mounted as /dev/lxd in the unpriv containers. If we want it to be accessible in there, it needs to be world readable on the host, so be 0755, not 0700.